### PR TITLE
ICMSLST-1409 Add preview and pre-sign versions of FA SIL licence.

### PR DESCRIPTION
**Pull request by MattHolmes123**
Other changes:

- Exclude "if TYPE_CHECKING" blocks from coverage report
- Add minimum coverage check to CI pipeline

### CODE REVIEW

File: web/tests/application_utils.py [hunk 1]
```diff
@@ -265,4 +349,7 @@ def submit_app(*, client: "Client", view_name: str, app_pk: int) -> None:
     form_data = {"confirmation": "I AGREE"}
     response = client.post(submit_url, form_data)
 
+    if response.status_code == 200:
+        print(response.context["errors"])
```
Comment (davidwtbuxton) [inline web/tests/application_utils.py:353]: Is this left over from earlier development work? Or should it be an assert?

**Review by mikeylockwood — APPROVED** [review]

File: web/utils/pdf/utils.py [hunk 2]
```diff
@@ -1,13 +1,26 @@
...
+    from web.domains.case._import.fa_dfl.models import DFLApplication
+    from web.domains.case._import.fa_oil.models import OpenIndividualLicenceApplication
+    from web.domains.case._import.fa_sil import models as sil_models
+    from web.models import ImportApplication
+
+    SILGoods = Union[
+        sil_models.SILGoodsSection1,
+        sil_models.SILGoodsSection2,
+        sil_models.SILGoodsSection5,
+        sil_models.SILGoodsSection582Obsolete,  # /PS-IGNORE
```
Comment (davidwtbuxton) [inline web/utils/pdf/utils.py:17]: Yeah, I'm guessing this is the post code PII checker, which does not do the Python tokenizing but instead scans the entire file as text.

**Review by davidwtbuxton — APPROVED** [review]

File: web/tests/application_utils.py (hunk 1 above)
Comment (MattHolmes123) [reply web/tests/application_utils.py:353]: Its just useful to see when the application has an error.
the next line asserts what it should do when its working correctly:
`assertRedirects(response, reverse("workbasket"), 302)`

Comment (MattHolmes123) [reply web/tests/application_utils.py:353]: I'm leaving it in as I always add it when I'm creating a new application type (to use in tests).

File: web/utils/pdf/utils.py (hunk 2 above)
Comment (MattHolmes123) [reply web/utils/pdf/utils.py:17]: :cry:

**Pull request merged**